src/lib.rs

```diff
@@ -389,6 +390,8 @@ impl Deref for ChallengeHandle<'_> {
 /// Create an [`Account`] with [`Account::create()`] or restore it from serialized data
 /// by passing deserialized [`AccountCredentials`] to [`Account::from_credentials()`].
 ///
+/// Alternatively, you can load an account using the private key using [`Account::loadgit ()`].
```

Comment:
I think some details shifted here and  `Account::loadgit()` isn't a fn?

Reply:
Lol, i guess my cli command got there by mistake